### PR TITLE
fix: separate context from Options

### PR DESCRIPTION
**Pull request by falconandy**

### CODE REVIEW

**Review by justin-iteratively — APPROVED** [review]
Looks good 👍

File: README.md [hunk 1]
```diff
@@ -297,7 +299,7 @@ There are [hygen](https://github.com/jondot/hygen) templates for creating a new
       added: packages/plugin-awesome-analytics/tsconfig.json
     ```
 
-3. Extend the ItlyPluginBase class in `packages/<name>/lib/index.ts`
+3. Extend the ItlyPlugin class in `packages/<name>/lib/index.ts`
```
Comment (justin-iteratively) [inline README.md:302]: ItlyPlugin => Itly 'Plugin'

File: _templates/sdk/new/lib-index.ts.t [hunk 2]
```diff
@@ -5,15 +5,14 @@ to: packages/sdk-<%= name %>/lib/index.ts
 import itlySdk, {
   ItlyOptions,
```
Comment (justin-iteratively) [inline _templates/sdk/new/lib-index.ts.t:6]: Can you please remove the `Itly` prefix from these imports. It's a legacy issue but would be nice to clean up.

File: README.md (hunk 1 above)
Comment (falconandy) [reply README.md]: Done

File: _templates/sdk/new/lib-index.ts.t (hunk 2 above)
Comment (falconandy) [reply _templates/sdk/new/lib-index.ts.t]: Done

**Pull request merged**